package.json

```diff
@@ -121,7 +121,27 @@
           "description": "Full path to C64 Debugger binary"
         }
       }
-    }
+    },
+    "keybindings": [
+      {
+        "command": "kickass-c64.build",
+        "key": "win+b",
+        "mac": "cmd+b",
```

Comment:
...and shift+cmd+b here?

Reply:
You're right. I somehow missed the default toggle side bar shortcut.